onnxruntime/python/tools/transformers/models/llama/README.md

```diff
@@ -372,3 +372,37 @@ python3 -m models.llama.benchmark_all \
     --num-runs 1000 \
     --timeout 60  # number of minutes before moving to the next benchmark
 ```
+
+# Mistral
```

Comment:
Can we add a table of contents at the beginning to show there's sections for both LLaMA and Mistral?

Reply:
Good idea! Included in latest commit/